docker/dev/Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM ubuntu:jammy
+FROM ubuntu:kinetic
```

Comment:
if we want to keep using ubuntu jammy, we'd have to drop clang-15.

Reply:
No problem upgrading.